app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,66 @@
+class LessonsController < ApplicationController
+  def index
+    @lessons = Lesson.where(user_id: current_user.id)
+  end
+
+  def show
+    @lesson = Lesson.find(params[:id])
+    @category = @lesson.category
+    @lesson_words = @lesson.lesson_words
+  end
+
+  def new
+    @lesson = Lesson.new
+    @category = Category.find(params[:category_id])
```

Comment:
báo lõio

Reply:
cái này thì user sẽ chọn category mới sang làm lesson được

Reply:
```suggestion
    @category = Category.find(params[:category_id])
```
bao loi neu k tim thay category